res/css/views/elements/_CopyableText.scss

```diff
@@ -22,7 +22,7 @@ limitations under the License.
     border: solid 1px $light-fg-color;
     margin-bottom: 10px;
     margin-top: 10px;
-    padding: 10px;
+    padding: 9px;
```

Comment:
Oh, alright. Thank you so much for your kind review. I'll surely keep these things in mind and try to come up with a better fix.
(Nothing so specific for 9px since it was the minimum modification I could think of to fix this. Thanks for your direction!)